go/runtime/host/multi/multi.go

```diff
@@ -143,32 +143,69 @@ func (agg *Aggregate) GetCapabilityTEE() (*node.CapabilityTEE, error) {
 	return agg.active.host.GetCapabilityTEE()
 }
 
+// shouldPropagateToNextVersion checks whether the given runtime request should also be propagated
+// to the next version that is pending activation.
+func shouldPropagateToNextVersion(body *protocol.Body) bool {
+	switch {
+	case body.RuntimeConsensusSyncRequest != nil:
+		// Consensus view of the next version should be up to date as otherwise signed attestations
+		// will be stale, resulting in them being rejected by the consensus layer.
+		return true
+	case body.RuntimeKeyManagerPolicyUpdateRequest != nil,
+		body.RuntimeKeyManagerStatusUpdateRequest != nil,
+		body.RuntimeKeyManagerQuotePolicyUpdateRequest != nil:
+		// Key manager updates should be propagated so that the runtime is ready when activated.
```

Comment:
This won't work as the new runtimes will fail to decode `body`. The reason is that old key managers support policy updates (and not status updates), while new ones support status updates (and not policy updates).

See: https://github.com/oasisprotocol/oasis-core/blob/master/go/runtime/registry/host.go#L649-L658

Solutions;
- Do not propagate `RuntimeKeyManagerPolicyUpdateRequest`.
- Propagate `RuntimeKeyManagerPolicyUpdateRequest` only if `nextRT.Features.KeyManagerStatusUpdates == false` (probably the same as previous one). 
- Update `host.go` (a lot of work).

Policy updates will be removed once we deploy 23.0.x.

Reply:
So the only problematic case that will be seen in practice (e.g. no downgrades) is where the active version is old (supporting policy updates) and the next version is new (supporting status updates). In this case the new runtime will not recognize the method and return an error (which will be logged but otherwise ignored).

This means that maybe we don't need to handle this explicitly?

Reply:
Discussed offline. The new runtime will not explode and will treat `RuntimeKeyManagerPolicyUpdateRequest` as invalid. So this can be marked as resolved.